jdisc-security-filters/pom.xml

```diff
@@ -23,6 +23,12 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>vespa-athenz</artifactId>
+            <version>${project.version}</version>
```

Comment:
is project.version the correct version to use?

Reply:
Yes